app/src/main/java/com/futurice/freesound/utils/TextUtils.java

```diff
@@ -44,6 +44,15 @@ public static boolean isNotNullOrEmpty(@Nullable final CharSequence chars) {
         return !isNullOrEmpty(chars);
     }
 
+    /**
+     * Evaluates if the given sequence is neither null nor empty.
+     *
+     * @return true if the parameter has at least one char, false otherwise.
+     */
+    public static boolean isNotEmpty(@Nullable final CharSequence chars) {
```

Comment:
Now I'm wondering what I asked you do to; this just seems to be the same as `isNotNullOrEmpty`.